app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>User details</h1>
\ No newline at end of file
+<h1>User details</h1>
+
+<div>
+  <div class="ml-3">
+  <h4 class="border ps-2 w-25 m-2 p-4 border-dark"> <%= @user.name %> </h4> 
+  </div>
+     <div class="border h-50 m-2 w-50 p-2 border-dark">
+      <span> bio</span> <br>
+      <%= @user.bio %>
+
+      </div><br>
+
+      <% @user.recent_posts.each do |post| %>
+       <div class="border h-50 w-50 p-2 m-2 border-dark">
+        <%= post.title %> <br> <br>
+        <%= post.text %>
+       </div>
+      <% end %>
+
+      <img src="<%= @user.photo %>"><br>
+        <%# <%= render partial: 'shared/user-card', locals: { user: user } %>
+
+<div>
```

Comment:
- [ ] Please consider following the given wireframe for your user show view, the comments and likes counter are missing as well as the layout and user photo and counter: 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/222302837-6f2cae44-a6f5-4365-bc6c-ea75b53cc002.png) | ![image](https://user-images.githubusercontent.com/77069011/222303508-1ff96d73-98a8-4c35-97ef-39b305c4fd3c.png) | 

Style is not needed at this point, but please at least lay out your elements as required! 🤓 🔧 

- [ ] Also you're missing the See all posts link to the `Post#index` view. Consider adding it. 🤓 🔧